src/examples/publish.yml

```diff
@@ -0,0 +1,16 @@
+description: Publish a new GitHub Release.
+
+usage:
+  version: 2.1
+  orbs:
+    github-release: civisanalytics/github-release@x.y.z
+  workflows:
+    your-workflow:
+      jobs:
+        - publish-github-release:
+            executor: github-release/github
+            steps:
+              - github-release/publish:
+                  current-version: a.b.c
```

Comment:
I'm a little confused as to why you'd want to hard-code the version into your circle config... Ohh, is it that you'd use this workflow when you're using circleci config itself to track the version of your code (and then publish new releases if and only if it changes?)

Reply:
ya, my thought is maybe someone doesn't have a version file and instead would want to publish versions through the config.